client/src/components/TODOList/AddNewTODO.js

```diff
@@ -120,73 +138,104 @@ const AddTODOItem = () => {
         size="lg"
         open={openAddTODO}
         handler={handleOpen}
-        className="addTODODialog shadow-none"
+        className="shadow-none"
       >
-        <Card className="addTODOForm">
-          <h2>Add TODO Item</h2>
-          <form onSubmit={handleSubmit}>
+        <Card className="m-4">
+          <h2 className="flex flex-row justify-center">Add TODO Item</h2>
+          <form
+            className="flex flex-col justify-evenly h-[30rem] overflow-y-auto"
+            onSubmit={handleSubmit}
+          >
             <div className="inputField">
-              <label htmlFor="add-title">TODO:</label>
               <Input
-                id="add-title"
                 name="title"
                 value={formData.title}
-                label="title"
+                label="Title"
                 onChange={handleInputChange}
               />
             </div>
-            <div className="inputField">
-              <label htmlFor="add-dueDate">Due Date:</label>
-              <DatePicker
-                id="add-dueDate"
-                className="bg-orange-200"
-                selected={formData.dueDate}
-                onChange={handleDueDateInput}
-              />
+            <div className="flex flex-row justify-between flex-wrap">
+              <div className="inputField mr-4">
+                <label htmlFor="add-startDate">Start Date:</label>
+                <DatePicker
```

Comment:
![image](https://github.com/awctw/study-dash/assets/56493743/5ebf8aba-4848-44d8-8521-2171b289c725)

Is the datepicker supposed to look like this? It has whitespace on the right, which the arrow for switching months uses, and the time selection is now on the bottom right. Tbh the react-datepicker for some reason looks like a scuffed version of input type="datetime-local":

![image](https://github.com/awctw/study-dash/assets/56493743/95bdeb4b-158b-4a0a-80ea-af9fc78930a1)

Reply:
yeah.... the time component that enables greater user flexibility with time input looks like that